src/vue/align-editor/align-editor-single-block.vue

```diff
@@ -7,7 +7,7 @@
       <template v-for = "token in alignTextData.tokens">
         <token
           v-if ="token.word"
-          :text-type = "textType" :text-word = "token"
+          :text-type = "textType" :text-word = "token" :key = "token.idWord"

```

Comment:
it is a tiny fix for @kirlats comment 
https://github.com/alpheios-project/alignment-editor-new/pull/10#discussion_r468583873